### PR TITLE
Feature/pelagos 4474 add profile page app

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: templates/Person/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html.twig'%}
+{% block title %}User Profile{% endblock %}
+
+{%block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('person-profile') }}
+
```
Comment (mickel1138) [inline templates/Person/index.html.twig:7]: line

File: assets/js/vue/pages/PersonProfile.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+        axiosInstance
+            .get(Routing.generate('pelagos_api_get_person') + "/" + this.personId)
+            .then(response => {
+                this.personProfileData = response.data;
+                this.showProfile = true;})
+            .catch(error => {
+                console.log(error);
+                this.showProfile = false;
+            });
```
Comment (mickel1138) [inline assets/js/vue/pages/PersonProfile.vue:72]: I don't know if this is a style choice, but we should pick either one tab over like this, or what we have done before, it have this one tab back.

Comment (praneethpr) [reply assets/js/vue/pages/PersonProfile.vue:72]: I looked up the Eslint rules for promise chaining, they say that new line for each chained call

File: src/Controller/Api/PersonController.php [hunk 3]
```diff
@@ -462,4 +462,38 @@ public function deleteAction(int $id)
...
+     * Returns a single person.
+     *
+     * @param Person $person The id of the Person.
+     *
+     * @Route(
+     *     "/api/person/{id}",
+     *     name="pelagos_api_get_person",
+     *     methods={"GET"},
+     *     defaults={"_format"="json"},
+     *     requirements={"id"="\d+"}
```
Comment (fightingtexasaggie) [inline src/Controller/Api/PersonController.php:477]: maybe we could make id optional, and if logged in and left null, it returns the json of the logged-in user..idk. Works well as-is too.

**Review by fightingtexasaggie — APPROVED** [review]
looks clean

**Review by mickel1138 — APPROVED** [review]
Double check to rights to the Anonymous image. Otherwise works!

**Pull request merged**